pom.xml

```diff
@@ -170,11 +170,13 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.11.0</version>
-          <!-- available from Java 9, but we went back to jdk8 for building
           <configuration>
-            <release>${java.version}</release>
+            <source>${java.version}</source>
+            <target>${java.version}</target>
+            <compilerArgs>
+              <arg>-parameters</arg>
```

Comment:
Is this working at your end? I tried your changes but it seems the `parameterNames` always end up being `null` despite the switch from `LocalVariableTableParameterNameDiscoverer` to `StandardReflectionParameterNameDiscoverer`. Wondering if there is anything else that needs to be refactored.

Reply:
I stand corrected. I forgot to recompile my classes with the new flag added. It now works!